x-pack/plugins/apm/public/utils/url.tsx

```diff
@@ -30,7 +30,7 @@ interface ViewMlJobArgs {
   serviceName: string;
   transactionType: string;
   location: any;
-  children?: any;
+  children?: React.ReactNode;
```

Comment:
Anywhere that we define children in an interface, I think it would be better to take that interface and pass it to React.SFC<PropsInterface> or React.ClassComponent<PropsInterface> and let React take care of the React-added props typings.

Reply:
Are you saying that if we use `React.SFC`, the interface can be changed to this?
```ts
 interface ViewMlJobArgs {
  serviceName: string;
  transactionType: string;
  location: any;
}
```

Reply:
Yeah that's my understanding

Reply:
Cool, will change